CaptainMumo/stackoverflow-lite
# #162625270 Correct email and password input type

#### What does this PR do?
Correct email and password input type on sign up and log in pages
#### Description of Task to be completed?
- Change email input type to 'email'
- Change password input type to 'password'
#### How should this be manually tested?
Clone this repo, open UI folder on text/code editor, then run signup.html or login.html on a browser and enter some characters in the password field. The input should be in bullets, not plain text as before.
#### Any background context you want to provide?
It is not possible to test the email validation as of now
#### What are the relevant pivotal tracker stories?
#162625270
#### Screenshots
Before fix
![image](https://user-images.githubusercontent.com/39964147/49904798-693ff880-fe7c-11e8-90ca-1b379a1595fe.png)
After fix
![image](https://user-images.githubusercontent.com/39964147/49904864-aefcc100-fe7c-11e8-8f7c-8d7638c0dd33.png)
